### ISSUE TITLE
CO2 concentration in CRTM

### ISSUE THREAD

**Issue by maroonaggie** [JCSDA/crtm#29]
Hello, I conducted a couple of clear-sky runs with CRTM-2.1.3 and the latest 2.4.0 version with the latest database. One thing I found is that, for both versions, the TOA BT simulations for all CO2 channels varies only if CO2 concentrations are in 360~400 ppmv range. I tested this for MODIS and GOES-R ABI. Is there a way we can go beyond the 360-400 boundary? Thanks!

**Comment by StegmannJCSDA** [comment]
Howdy @maroonaggie. Yes, that is roughly the range of the CO2 training profiles we are using, so we are limited to that.
In order to extend the CO2 range we would need a representative dataset of atmospheric profiles with a broader CO2 range than the current one to re-train the desired CRTM instrument coefficients.

**Comment by StegmannJCSDA** [comment]
Closing.

**Comment by maroonaggie** [comment]
Thanks! Do you have plan to extend the CO2 range in the near future?

**Comment by StegmannJCSDA** [comment]
@maroonaggie it's certainly interesting but changing the CO2 predictors will involve a lot of work and including more extreme ranges may possibly deteriorate the CRTM performance for the more common CO2 ranges.
On the other, we also need to make efforts to take the steadily increasing global CO2 concentration into account.

tl;dr: Not in the near future, but it needs to be part of our long-term agenda.

**Comment by maroonaggie** [comment]
Gotta, thanks.
